platform/engineering/code_review_guidelines.md

```diff
@@ -8,6 +8,21 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
 
 We expect developers to be familiar with GitHub and the pull request review process, but want to be clear about our norms to reduce confusion and help code flow through the process as quickly and reliably as possible.
 
+## Code owners on vets-website
+
+VSP uses GitHub's [code owners](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) feature to trigger required reviews. A code owner is a GitHub [organization](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-organizations) that must approve a change before it can be merged. 
+
+By default, the [frontend review group](https://github.com/orgs/department-of-veterans-affairs/teams/frontend-review-group) is the code owner of vets-website. VSA teams can optionally create a GitHub team under the [Department of Veteran Affairs GitHub organization](https://github.com/orgs/department-of-veterans-affairs/teams) and assign that team as a code owner of application [folders](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications). **This will remove the frontend review group as a required reviewer when changes are made to those folders.** 
+
+The code owners configuration can be found [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/CODEOWNERS). After creating your GitHub team, follow the conventions in the file to assign your team to an application folder. You should understand the following when making changes to the code owners configuration: 
+
+- The **last matching rule** that matches all files in a change **takes precedence**.
+- Application code should be structured to **require as few rules as possible**. 
+- Only one required reviewer at a time can be triggered by code owners rules.
+- Code owners rules are applied to every push after a PR is created. Required reviewers can be removed during this process but **GitHub does not remove the front end review group once it has been added to a PR** as a required reviewer. 
```

Comment:
```suggestion
- Code owners rules are applied to every push after a PR is created. Required reviewers can be removed during this process but **GitHub does not remove the frontend review group once it has been added to a PR** as a required reviewer. 
```
We probably should decide `frontend` vs `front end`, but you've been using `frontend` the rest of the time in this section. :man_shrugging: